src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexNode.cs

```diff
@@ -889,6 +889,20 @@ private RegexNode ReduceSet()
                     RegexNodeKind.Notonelazy;
             }
 
+            // Normalize some well-known sets
+            switch (Str)
+            {
+                // Different ways of saying "match anything"
+                case RegexCharClass.WordNotWordClass:
```

Comment:
we mostly don't have tests for `[\d\D]` specifically - worth adding a test for each? that would verify that they still have correct behavior.

Reply:
Added